js/src/admin/components/DashboardWidget.js

```diff
@@ -20,6 +20,6 @@ export default class DashboardWidget extends Component {
    * @return {VirtualElement}
    */
   content() {
-    return [];
+    return null;
```

Comment:
This one could be breaking if someone was extending this method. But that's probably fine since I don't think this is public API (it's not an itemlist).

Reply:
Yeah, IMO extending functions like this is risky and we could break it at any time. On the other hand, everything should be itemlists!